samples/client/petstore/csharp-netcore/OpenAPIClient-generichost-net6.0-nrt/src/Org.OpenAPITools/Model/Capitalization.cs

```diff
@@ -0,0 +1,239 @@
+// <auto-generated>
```

Comment:
@devhl-labs FYI. This PR "fixes" the naming of some C# generichost models.

one way to fallback is to use the modelNameMappings option

Reply:
Why is this file new with no edits to the yaml?

Reply:
my bad. these files were incorrectly created and now removed